labkey-client-api/src/org/labkey/remoteapi/security/StopImpersonatingCommand.java

```diff
@@ -10,6 +10,6 @@ public class StopImpersonatingCommand extends PostCommand<CommandResponse>
 {
     public StopImpersonatingCommand()
     {
-        super("login", "stopImpersonating.api");
+        super("login", "stopImpersonatingApi.api");
```

Comment:
I'm not sure what advantage this has. It seems to just break backward compatibility.